ansible/files/ocp/compute-node/computenodeopenstack.yaml

```diff
@@ -19,19 +19,9 @@ spec:
     commonConfigMap: common-config
     ospSecrets: osp-secrets
   network:
-    nic: "enp6s0"
-    bridgeMappings: ""
-    mechanismDrivers: ""
-    servicePlugins: ""
+    nic: "enp2s0"
```

Comment:
does the PR needs rebase? I already changed this on another PR that is already merged: https://github.com/openstack-k8s-operators/dev-tools/pull/56/files

Reply:
yes, I added that local modified file by accident